### PR TITLE
Add handling of no matching between Harvest and Slack

### PR DESCRIPTION
**Pull request by chubberlisk**

### CODE REVIEW

**Comment by dushan-madetech** [comment]
looks good to me 👍

File: CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs [hunk 1]
```diff
@@ -274,5 +274,82 @@ public void CanGetLateADeveloper(int harvestUserId, string slackUserId)
...
+                    new DateTimeOffset(
+                        new DateTime(2019, 03, 01, 10, 30, 0)
+                    )
+                );
+                
+                var getDevelopers = new GetLateDevelopers(_slackGatewayStub, _harvestGatewayStub, _harvestGatewayStub, clock);
+                
+                var response = getDevelopers.Execute();
+
+                response.Developers.First().Should().Be("U8723");
```
Comment (C-gyorfi) [inline CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs]: Is this single test enough? What if we have 0 matches?

Comment (chubberlisk) [reply CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs]: I think this is a valid point. We can add more test cases.

Comment (yusufsheiqh) [reply CryptoTechReminderSystem.Test/UseCase/GetLateDevelopersTests.cs]: Resolved in https://github.com/chubberlisk/crypto_tech/pull/27/commits/1b920010a845e10b1c3c6419ffb7bd2a7fb9eb44.

**Pull request merged**